### PR TITLE
Fall back to navigator.clipboard if navigator.share isn't supported.

### LINKED ISSUE #24: use clipboard API if web-share is not supported
Web Share is currently not supported on most desktop platforms. As an alternative, we can fall back to the [Clipboard API](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard_API).

This is a follow-up to Issue #1 for after https://github.com/rknoll/draw/pull/21 lands.

### PR DESCRIPTION
**Pull request by glennhartmann**
Fixes https://github.com/rknoll/draw/issues/24.

### CODE REVIEW

**Comment by glennhartmann** [comment]
Untitled

File: client/components/ShareButton.js [hunk 1]
```diff
@@ -21,33 +22,70 @@ const useStyles = ((canShare) => makeStyles((theme) => ({
 export default () => {
   const [state, setState] = useState({showSnackbar: false});
   const shareData = {url: window.location.href};
-  const classes = useStyles(navigator.canShare && navigator.canShare(shareData))();
+  const canShare = navigator.canShare && navigator.canShare(shareData);
+  const classes = useStyles(
+    canShare || (navigator.clipboard && navigator.clipboard.writeText))();
```
Comment (rknoll) [inline client/components/ShareButton.js]: nit: extract the clipboard stuff into a new variable?
```js
const canShare = navigator.canShare && navigator.canShare(shareData);
const canCopy = navigator.clipboard && navigator.clipboard.writeText;

const classes = useStyles(canShare || canCopy)();
```

**Review by rknoll — APPROVED** [review]
Nice!

File: client/components/ShareButton.js (hunk 1 above)
Comment (glennhartmann) [reply client/components/ShareButton.js]: Done.

**Pull request merged**